overseerr-api.yml

```diff
@@ -2560,6 +2737,36 @@ paths:
                   type: string
               required:
                 - authToken
+  /auth/jellyfin:
+    post:
+      summary: Sign in using a Jellyfin username and password
+      description: Takes the user's username and password to log the user in. Generates a session cookie for use in further requests. If the user does not exist, and there are no other users, then a user will be created with full admin privileges. If a user logs in with access to the main Plex server, they will also have an account created, but without any permissions.
```

Comment:
This description needs to be updated to match what the behaviour is for Jellyfin